spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigMapPropertySourceLocator.java

```diff
@@ -45,11 +45,11 @@ public Fabric8ConfigMapPropertySourceLocator(KubernetesClient client, ConfigMapC
 	}
 
 	@Override
```

Comment:
- instead of `String name` use `String applicationName`; it's a minor change, but reading the code becomes a lot more easy (from personal experience at least)
- I've extracted this:

```
String namespaceName = getApplicationNamespace(this.client, normalizedSource.getNamespace(),
				configurationTarget);
```

to a separate call. Again, for the reading purposes only.